src/lib/utils/session-token.ts

```diff
@@ -21,8 +24,9 @@ export async function createSessionToken(
     .sign(secret);
 }
 
-export const getSessionToken = (req: NextRequest): string | null =>
-  req.cookies.get(COOKIES.session)?.value ?? null;
+export async function getSessionToken(): Promise<string | null> {
+  return (await cookies()).get(SESSION_COOKIE)?.value || null;
```

Comment:
хотите константу для куки? их есть у нас (в @/lib/constants/cookie и мы их уже импротировали)

```suggestion
  return (await cookies()).get(COOKIES.session)?.value || null;
```